java/pom.xml

```diff
@@ -92,7 +92,7 @@
         <artifactId>maven-surefire-plugin</artifactId>
         <version>2.12</version>
         <configuration>
-          <forkMode>never</forkMode>
+          <forkCount>0</forkCount>
```

Comment:
forkMode is deprecated, was giving some ambiguous warning about how this is incompatible with the default version of some other flag we're not even setting.